gridstatus/tests/test_spp.py

```diff
@@ -219,6 +219,41 @@ def test_get_operating_reserves_latest(self):
         assert len(df) > 0
         assert df.columns.tolist() == self.OPERATING_RESERVES_COLUMNS
 
+    DAY_AHEAD_MARGINAL_CLEARING_PRICES_COLUMNS = [
+        "Time",
+        "Interval Start",
+        "Interval End",
+        "Interval",
+        "Reserve Zone",
+        "Reg_Up_Cleared",
+        "Reg_Dn_Cleared",
+        "Ramp_Up_Cleared",
+        "Ramp_Dn_Cleared",
+        "Unc_Up_Cleared",
+        "Spin_Cleared",
+        "Supp_Cleared",
+    ]
+
+    def test_get_day_ahead_marginal_clearing_prices(self):
```

Comment:
it's hourly data, but released in daily files. probably best to do tests that checkout requesting one day and multiple days

Reply:
Thanks, updated the tests.